### PR TITLE
Feature/database scheme

### PR DESCRIPTION
**Pull request by Pavliuk13**
database schema created

### CODE REVIEW

**Review by tatianahutii — CHANGES_REQUESTED** [review]
Please review the changed/added files
We don't need package-lock files and the second yarn.lock file

File: README.md [hunk 1]
```diff
@@ -38,10 +38,141 @@ _Tip: If you want to connect to the specific service outside of docker, then use
...
+  UserTeam {
+      int UserId
+      int RoleId
+      int TeamId
+      int Status
+      datetime CreatedAt
+      datetime UpdatedAt
+  }
+
+  Role {
```
Comment (tatianahutii) [inline README.md]: Could you please remove the Role table?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Erbidium) [reply README.md]: Removed

**Comment by Erbidium** [comment]
> Please review the changed/added files We don't need package-lock files and the second yarn.lock file

Removed unnecessary files)

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**